tests_integration/__tests__/__snapshots__/arg-parsing.js.snap

```diff
@@ -22,7 +22,7 @@ exports[`deprecated option values are warned (stdout) 1`] = `
 exports[`deprecated option values are warned (write) 1`] = `Array []`;
 
 exports[`deprecated options are warned (stderr) 1`] = `
-"\`--flow-parser\` is deprecated. Use \`--parser flow\` instead.
+"warn \`--flow-parser\` is deprecated. Use \`--parser flow\` instead.
```

Comment:
Could we change it to 

```
warn: `--flow-parser` is deprecated...
```

or

```
[warn] `--flow-parser` is deprecated...
```

Reply:
![image](https://user-images.githubusercontent.com/8341033/31316656-3f0b39cc-abf7-11e7-85f1-99295990ff8a.png)

`[xxx]` looks better to me.

Reply:
Looks good. It was just but strange without punctuation and without color.